adapters/amqp-vertx/pom.xml

```diff
@@ -50,8 +50,14 @@
             <configuration>
               <images>
                 <image>
+                  <name>
+                      ${docker.repository}/${project.artifactId}:${project.version}
```

Comment:
we currently define the name for all images in the parent POM's `<build>` section:
```
        <plugin>
          <groupId>io.fabric8</groupId>
          <artifactId>docker-maven-plugin</artifactId>
          <version>0.28.0</version>
          <executions>
            <execution>
              <id>build_images</id>
              <phase>package</phase>
              <goals>
                <goal>build</goal>
              </goals>
            </execution>
          </executions>
          <configuration>
            <images>
              <image>
                <name>${docker.image.org-name}/%a:%v</name>
                <build>
                  <maintainer>The Eclipse Hono project</maintainer>
                  <labels>
                    <project>Eclipse Hono</project>
                  </labels>
                  <tags>
                    <tag>latest</tag>
                  </tags>
                </build>
              </image>
            </images>
          </configuration>
        </plugin>
```
So, you can set the *docker.image.org-name* property to e.g. `aloxy` instead of using the default `eclipse` repository. You can then use the `docker.push.registry` property to set the host:port of the registry you want the images to be pushed when running the `push` goal. Based on that, IMHO the whole `<name>` element can be removed. See http://dmp.fabric8.io/#docker:push for details ...

Reply:
You are right. changed it